### PR TITLE
add av1 decoder support

### PR DESCRIPTION
**Pull request by yezhan10**
add dav1d av1 decoder

### CODE REVIEW

File: src/av1/AV1decoder.cpp [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+		case AVCOL_RANGE_JPEG: 
+			//2^n-1   "JPEG" YUV ranges
+			videoBuffer->SetColorRange(VideoBuffer::ColorRange::Full);
+			break;
+		default:
+			//Unknown
+			videoBuffer->SetColorRange(VideoBuffer::ColorRange::Unknown);
+	}
+	
+	// TODO: color space?
```
Comment (phil-moss-dolby) [inline src/av1/AV1decoder.cpp:130]: Should be the same color spaces as h264, in addition AV1 can handle HDR10 and HDR metadata. I just don't know if we're adding those cause I don't see them in h265decoder

File: src/VideoCodecFactory.cpp [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #include "vp9/VP9Encoder.h"
 #include "jpeg/JPEGEncoder.h"
 #include "webp/WEBPEncoder.h"
+#include "av1/AV1decoder.h"
```
Comment (murillo128) [inline src/VideoCodecFactory.cpp:12]: Pleas rename

```suggestion
#include "av1/AV1Decoder.h"
```

**Review by murillo128 — APPROVED** [review]
lgtm, but please rename the AV1Decoder files to match the class name case

File: src/av1/AV1decoder.cpp (hunk 1 above)
Comment (yezhan10) [reply src/av1/AV1decoder.cpp]: new commit has color space set as the same as h264

**Review by murillo128 — APPROVED** [review]
replace space idents on test file by tabs, otherwise lgtm. Good job!

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -20,6 +20,9 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install --fix-missing -y libavcodec-dev libx264-dev libx265-dev libavutil-dev libvpx-dev libswscale-dev libswresample-dev libavformat-dev libssl-dev libavfilter-dev
+        sudo wget https://ftp.debian.org/debian/pool/main/a/aom/libaom3_3.9.1-1_amd64.deb
```
Comment (murillo128) [inline .github/workflows/release.yml:23]: I have merged the AV1 encoder support, so we can remove this from here

**Pull request merged**